engine/src/test/java/io/zeebe/engine/processor/workflow/job/FailJobTest.java

```diff
@@ -193,20 +193,6 @@ public void shouldRejectFailIfJobAlreadyFailed() {
     assertThat(jobRecord.getRejectionReason()).contains("it is in state 'FAILED'");
   }
 
-  @Test
-  public void shouldRejectFailIfJobCreated() {
-    // given
-    final Record<JobRecordValue> job = ENGINE.createJob(jobType, PROCESS_ID);
-
-    // when
-    final Record<JobRecordValue> jobRecord =
```

Comment:
So we should now be able to fail a job without activating it. Could we re-add this one and assert that it is failed correctly.

Reply:
I re-added it with some modifications and moved the test case up so its together with the other successful command cases.